analyses/doublet-detection/README.md

```diff
@@ -28,15 +28,24 @@ There are two portions of this module, which can be run as follows:
 This analysis runs several doublet detection methods and assesses and compares their performance.
   ```sh
   ./run_doublet-detection-benchmarking.sh
+
+  # to run in test mode using subsetted datasets:
+  test=1 ./run_doublet-detection-benchmarking.sh
+
+  # the script uses 4 cores by default. Use the cores argument to customize:
+  cores=2 ./run_doublet-detection-benchmarking.sh
   ```
 
 - ScPCA analysis: Detect doublets in ScPCA data using `scDblFinder` for a given project id (`SCPCPXXXXXX`).
   - The conda environment is not needed for this analysis, since it is only R-based
   ```sh
   ./run_doublet-detection-scpca.sh {scpca project id}
 
-  # for example, detect doublets on all libraries in ScPCA project SCPCP000001:
-  ./run_doublet-detection-scpca.sh SCPCP000001
+  # the script uses 4 cores by default. Use the cores argument to customize:
+  cores=2 ./run_doublet-detection-scpca.sh {scpca project id}
+
+  # for example, detect doublets on all libraries in ScPCA project SCPCP000001 with 6 cores:
+  cores=6 ./run_doublet-detection-scpca.sh SCPCP000001
```

Comment:
I think you could simplify this to one line and just show the last example, but up to you.